newIDE/app/src/guidelines/index.js

```diff
@@ -325,14 +311,14 @@ export default class GuidelinePopOver extends PureComponent<Props, State> {
                         <Line>{closeOrWikipage}</Line>
                         <Line alignItems="center">
                           <Typography style={{ marginRight: 10 }}>
-                            {this.state.index + 1} of {this.state.indexMax + 1}
+                            {this.state.index + 1} <Trans>of</Trans> {this.state.indexMax + 1}
```

Comment:
You can actually use variables in `<Trans>` :) `<Trans>{this.state.index + 1} of {this.state.indexMax + 1}</Trans>`

js-lingui will generate a translation of the form `{0} of {1}` - which is useful for languages where the order of words is reversed.